google-map-marker.html

```diff
@@ -396,6 +396,7 @@
           this.info.open(this.map, this.marker);
           this.fire('google-map-marker-open');
         } else {
+          this.content = this.info.getContent();
```

Comment:
I'll add a comment to the code if you think I'm still on the right track and this remains applicable.